src/util/useCheckCanEdit.ts

```diff
@@ -4,12 +4,16 @@ import { Acl } from "@madie/madie-models/dist/Measure";
 const useCheckUserCanEdit = (
   createdBy: string,
   acls: Array<Acl>,
-  draft: boolean = true
+  draft: boolean = true,
+  editTestsOnVersionedMeasures?: boolean
 ): boolean => {
   const { getUserName } = useOktaTokens();
   const userName = getUserName();
   // versioned measures/libraries are always uneditable.
   if (!draft) {
+    if (editTestsOnVersionedMeasures) {
+      return true;
```

Comment:
Needs to consider ownership/shared permissions. Story ACs call out "user with edit access":
> User with edit access to the measure can add a new test case to a versioned measures
User with edit access to the measure can clone existing test cases on a versioned measures
User with edit access to the measure can import test cases to a versioned measure (import as new test cases)

Reply:
thank you, I missed that scenario I have fixed it now